<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>71</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>TATSSI</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDownloaders">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionDownloaders"/>
    <addaction name="actionImporter"/>
   </widget>
   <widget class="QMenu" name="menuQuality">
    <property name="title">
     <string>Quality</string>
    </property>
    <addaction name="actionAnalytics"/>
   </widget>
   <widget class="QMenu" name="menuTime_Series">
    <property name="title">
     <string>Time Series</string>
    </property>
    <addaction name="actionGenerator"/>
    <addaction name="actionSmoothing"/>
    <addaction name="actionAnalysis"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionTATSSI"/>
   </widget>
   <addaction name="menuDownloaders"/>
   <addaction name="menuTime_Series"/>
   <addaction name="menuQuality"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTranslator">
   <property name="text">
    <string>Translator</string>
   </property>
  </action>
  <action name="actionDownloaders">
   <property name="text">
    <string>Downloaders</string>
   </property>
  </action>
  <action name="actionAnalytics">
   <property name="text">
    <string>Analytics</string>
   </property>
  </action>
  <action name="actionSmoothing">
   <property name="text">
    <string>Smoothing</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="text">
    <string>Analysis</string>
   </property>
  </action>
  <action name="actionGenerator">
   <property name="text">
    <string>Generator</string>
   </property>
  </action>
  <action name="actionTATSSI">
   <property name="text">
    <string>TATSSI</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="text">
    <string>GeoTIFF Importer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
